backend/apps/oauth/views.py

```
@@ -0,0 +1,364 @@
+        })
+
+
+class JWKSView(views.APIView):
+    permission_classes = [permissions.AllowAny]
+
+    def get(self, request):
+        return Response(get_jwks())
+
+
+class IntrospectionView(views.APIView):
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)